packages/fxa-settings/src/pages/Signin/SigninBounced/index.stories.tsx

```diff
@@ -3,30 +3,33 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import SigninBounced, { SigninBouncedProps } from '.';
-import AppLayout from '../../../components/AppLayout';
+import SigninBounced from '.';
 import { LocationProvider } from '@reach/router';
 import { Meta } from '@storybook/react';
 import { withLocalization } from '../../../../.storybook/decorators';
+import { MOCK_ACCOUNT } from '../../../models/mocks';
 
 export default {
   title: 'Pages/Signin/SigninBounced',
   component: SigninBounced,
   decorators: [withLocalization],
 } as Meta;
 
-const ComponentWithRouter = (props: SigninBouncedProps) => (
+export const Default = () => (
   <LocationProvider>
-    <AppLayout>
-      <SigninBounced email={props.email} canGoBack={props.canGoBack} />
-    </AppLayout>
+    <SigninBounced
+      emailLookupComplete={true}
```

Comment:
~nit: I think we can drop the `{true}` for boolean props.~
Meh, it's a `*.stories.*` file, maybe we don't care too much. 🤷